ls1intum/Artemis
# `Programming exercises`: Add automatic expanding if there is only one task in the grading configuration

### Checklist
#### General



### Motivation and Context


Small improvement for instructors for easier interaction with the grading configuration.

### Description

Adds automatic expanding if there is only one task

### Steps for Testing

Prerequisites:
- 1 Instructor

1. Create a programming exercise without tasks by deleting the entire problem statement.
1. Go to the grading configuration e.g. `ARTEMISURL/course-management/1/programming-exercises/3/grading/test-cases`
1. Check if the `Not assigned to task` task is expanded.

### Review Progress






#### Performance Review
#### Code Review
#### Manual Tests
#### Exam Mode Test

### Test Coverage





### Screenshots
